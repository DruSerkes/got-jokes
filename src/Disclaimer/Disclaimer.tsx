import * as React from 'react';

export const Disclaimer = () => {
    return (
        <p>
            <small className="mb-4">
                <strong>Disclaimer:</strong> Jokes are being served via <a href="https://v2.jokeapi.dev/" target="_blank" rel="noopener noreferrer">JokeAPI</a>
                <br />
                Please report any offensive, racist, sexist, or generally horrible / unfunny content
            </small>
        </p>
    )
};