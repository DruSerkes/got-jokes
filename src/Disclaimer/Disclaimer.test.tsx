import * as React from 'react';
import { render } from '@testing-library/react';
import { Disclaimer } from './Disclaimer';

describe('Disclaimer tests', () => {

    it('should render and match snapshot', () => {
        const { asFragment } = render(<Disclaimer />);
        expect(asFragment()).toMatchSnapshot();
    });

}); 