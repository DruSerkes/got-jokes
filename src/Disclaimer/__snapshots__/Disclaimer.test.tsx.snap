// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disclaimer tests should render and match snapshot 1`] = `
<DocumentFragment>
  <small
    class="mb-4"
  >
    <strong>
      Disclaimer:
    </strong>
     Jokes are being served via 
    <a
      href="https://v2.jokeapi.dev/"
      rel="noopener noreferrer"
      target="_blank"
    >
      JokeAPI
    </a>
    <br />
    Please report any offensive, racist, sexist, or generally horrible / unfunny content
  </small>
</DocumentFragment>
`;
